src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.tsx

```diff
@@ -0,0 +1,142 @@
+  },
+  description: {
+    id: 'dashboard.ActionsPage.StakingWidget.description',
+    defaultMessage: `Stake is returned if the motion passes. If there is a dispute, and the motion loses, part or all of your stake will be lost.`,
+  },
+  stakeButton: {
+    id: 'dashboard.ActionsPage.StakingWidget.stakeButton',
+    defaultMessage: 'Stake',
+  },
+  objectButton: {
+    id: 'dashboard.ActionsPage.StakingWidget.objectButton',
+    defaultMessage: 'Object',
+  },
+});
+
+const StakingWidget = ({
+  motionId,
+  motionDomainId,
+  rootHash,
+  colonyAddress,
+  tokenDecimals,
+}: Props) => {
+  const { walletAddress } = useLoggedInUser();
+  const { data, loading } = useStakeMotionLimitsQuery({
+    variables: {
+      colonyAddress,
+      userAddress: walletAddress,
+      motionId,
+      tokenDecimals,
+    },
+  });
+  const { data: userColonyAddressesData } = useUserColonyAddressesQuery({
+    variables: { address: walletAddress },
+  });
+  const validationSchema = yup.object().shape({
+    amount: yup.number().required(),
+  });
+
+  const transform = useCallback(
+    pipe(
+      mapPayload(({ amount }) => {
+        return {
+          amount: bigNumberify(amount).mul(bigNumberify(10).pow(tokenDecimals)),
+          userAddress: walletAddress,
+          rootHash,
+          colonyAddress,
+          motionId: bigNumberify(motionId),
+          motionDomainId,
+          vote: bigNumberify(1),
+        };
+      }),
+    ),
+    [walletAddress, colonyAddress, motionId],
+  );
+
+  if (loading || !data?.stakeMotionLimits || !userColonyAddressesData?.user) {
+    return null;
+  }
+
+  const {
+    user: { colonyAddresses },
+  } = userColonyAddressesData;
+
+  const isSubscribed = (colonyAddresses || []).includes(colonyAddress);
+  const { minStake, maxStake, requiredStake } = data.stakeMotionLimits;
+
+  return (
+    <ActionForm
+      initialValues={{
+        amount: minStake,
+      }}
+      validationSchema={validationSchema}
+      submit={ActionTypes.MOTION_STAKE}
+      error={ActionTypes.MOTION_STAKE_ERROR}
+      success={ActionTypes.MOTION_STAKE_SUCCESS}
+      transform={transform}
+    >
+      {({ values, isValid }) => (
+        <div className={styles.wrapper}>
+          <Heading text={MSG.title} className={styles.title} />
+          <p className={styles.description}>
+            <FormattedMessage {...MSG.description} />
+          </p>
+          <span className={styles.amount}>{values.amount}</span>
+          <Slider
+            name="amount"
+            value={values.amount}
+            min={minStake}
+            max={requiredStake}
+            limit={maxStake}
+          />
+          <div className={styles.buttonGroup}>
+            <Button
+              type="submit"
+              disabled={!isValid || !isSubscribed}
+              text={MSG.stakeButton}
+            />
+            <Button appearance={{ theme: 'pink' }} text={MSG.objectButton} />
+          </div>
```

Comment:
The slider and buttons are not aligned properly on the same vertical line

![Screenshot from 2021-04-16 15-30-20](https://user-images.githubusercontent.com/1193222/115024415-bdce6500-9ec8-11eb-8a4e-c7dd4f5ca2cf.png)